app/scripts/components/common/map/style/button.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+import { createButtonStyles } from '@devseed-ui/button';
+import { themeVal } from '@devseed-ui/theme-provider';
+
+import { TipButton } from '$components/common/tip-button';
+
+// Why you ask? Very well:
+// Mapbox's css has an instruction that sets the hover color for buttons to
+// near black. The only way to override it is to increase the specificity and
+// we need the button functions to get the correct color.
+// The infamous instruction:
+// .mapboxgl-ctrl button:not(:disabled):hover {
+//   background-color: rgba(0, 0, 0, 0.05);
+// }
+
+export const SelectorButton = styled(TipButton)`
+  &&& {
```

Comment:
❓ : Curious - what is this `&&&` ?

Reply:
I think I had the exact same question when I first saw it! It is to bump up the specificity of the selector so we can override: https://styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity (Maybe this should go into the comment ^-^;)

Reply:
I changed the comment. I hope that the new comment actually answers the questions the code-reviewers would have. I found this comment helpful Especially because Mapbox doesn't offer an official doc about how to override their styles 
- Does the new comment help better?
- Is there any better way to clearly indicate this maneuver for styles?